cmd/commands/run.go

```diff
@@ -289,6 +289,8 @@ func runFunction(ctx context.Context, fn function.Function, opts runFunctionOpts
 	log := logger.Buffered(buf)
 	ctx = logger.With(ctx, *log)
 
+	fmt.Println("RUNNING")
```

Comment:
Is this left from debugging or is this intentional? Just saw it outside of bubbletea :)